src/lib/usbmux.ml

```diff
@@ -462,6 +466,7 @@ module Relay = struct
     |> Lwt.async
 
   let rec make_tunnels
+      ?(bind_host=None)
```

Comment:
@mdolinin well it means that then the different makeup is giving a type error somewhere else, so go to line 551 and see how you're using it there. 

I can accept the previous commit cause you did implement it, but its good as well to get better at OCaml and you are doing well!